test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchRegression

```diff
@@ -5,5 +5,6 @@ BIN_DIR=${HOME_DIR}/bin
 LOG_DIR=${HOME_DIR}/logs
 
 git clone https://github.com/pytorch/examples.git ${HOME_DIR}/artifacts/examples
+sed -i "s/param.add_(-0.1 \* param.grad)/param.data.add_(-0.1 \* param.grad)/g" ${HOME_DIR}/artifacts/examples/regression/main.py # temporary patch for isssue https://github.com/pytorch/examples/issues/782
```

Comment:
Why this change?

Reply:
There is a issue with this test. The PT example repo fixed a similar issue, but seems forgot to fix this file. https://github.com/pytorch/examples/pull/779

I opened a PR for their repo but did not have response yet. This is a temp approach to make the test pass.